lib/functions.ts

```diff
@@ -0,0 +1,7 @@
+export function getPlayerName() {
```

Comment:
Maybe find a better name than `functions` for this file ;)

Reply:
Yeah, maybe when I add more functions the purpose will become clearer. It's a placeholder for now.
If you have any suggestions, please share them :)

Reply:
What about simply `localStorage`, when you add more functions simply place them in a new file so that you have some coherence between the functions.